### PR TITLE
Feature/gyms

### PR DESCRIPTION
**Pull request by KathLevi**
Fixed some merge issues as well as added a few more minor fixes for people w/o a profile

### CODE REVIEW

File: lib/screens/current_set.dart [hunk 1]
```diff
@@ -105,14 +119,41 @@ class CurrentSetState extends State<CurrentSet> {
...
+                    padding: const EdgeInsets.symmetric(vertical: 20.0),
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: <Widget>[
+                        //todo: add route routing
+                        ratingWidget(
+                            function: () => print('top rope'),
+                            route: topRopeRoute),
+                        ratingWidget(
+                            function: () => print('bounder'),
```
Comment (Monkey350) [inline lib/screens/current_set.dart:152]: bounder?

Comment (KathLevi) [reply lib/screens/current_set.dart:152]: It's just a print statement to hold the palce until the actual function is put in. I don't think it matters

Comment (swordensen) [reply lib/screens/current_set.dart:152]: PRINT THE BOUNDER!!!

**Review by swordensen — APPROVED** [review]
bounder bounder bounder bounder bounder bounder

**Pull request merged**